crates/emitter/src/opcode_info.rs

```diff
@@ -1,7 +1,35 @@
+pub fn get_async_function_resolve_kind() -> &'static str {
+    include_str!("copy/AsyncFunctionResolveKind.h")
+}
```

Comment:
Doing so would be a binary size issue, which would prevent us from shipping jsparagus. This is not a blocker nor a priority for now, but we should file an issue about it.

Maybe there is a better way to do it which does not involve the source file content in the binary.

Reply:
In the long term, all those definition should be moved to Rust code, or somewhere more accessible from both C++ and Rust, so this issue should go away.

meanwhile, I think we can move `opcode_info` under optional feature, and enable it only while building SpiderMonkey (in build.rs, not in JS shell)